pkg/restapi/vc/operation/persistence.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
```

Comment:
How about naming this file "recorder" instead? "persistence" to me sounds like it's a storage file. And persistence usually means storage that doesn't go away when power is off, and it's possible to use memstore here https://searchstorage.techtarget.com/definition/Persistent-storage

Reply:
(see https://github.com/trustbloc/edge-service/pull/10#discussion_r368071964 before acting on this comment)

Reply:
done